docs/docs/guides/Web3_Utils_module/Mastering_web3-utils.md

```diff
@@ -0,0 +1,226 @@
+# Mastering web3.js Utils
+
+## Introduction
+
+## Imports
+
+There are three different way to import utils package.
+
+1. Import the entire web3 module and initialize a provider:
+
+    `const [ Web3 ] = require("web3"); ` 
+
+    then inialiazing a provider with 
+    `const web3 = new Web3("");`
+
+    To be able to access the package you need to type
+    `web3.utils.toWei()`
+
+
+2. Import only the utils package from web3:
+
+    `const { utils } = require("web3");` 
+
+    Then you are also able to use any function like
+    `utils.toWei()`
+
+3. Import a specific function from the utils package: 
+
+    `const { toWei } = require("web3-utils");`
+    
+    then you'll be able to use it here `toWei();`
+
+
+## Random Bytes
+
+There are two way to generate Random Bytes.
+
+1. The first function that is knownn as Random Bytes, it receives a parameter of the size of bytes you want to generate and it will return the bytes in an array format.
+
+    array format `console.log(utils.randomBytes());`
+
+2. The second function is known as Randon Hex also requires a parameter of the size of bytes and it will return the btytes in Hexadecimal format
```

Comment:
```suggestion
2. The second function is known as Random Hex also requires a parameter of the size of bytes and it will return the bytes in Hexadecimal format
```